modules/dxp/apps/commerce/commerce-organization-web/source-formatter-suppressions.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0"?>
+
+<suppressions>
+	<source-check>
+		<suppress checks="LanguageKeysCheck" files="src/main/java/com/liferay/commerce/organization/web/internal/frontend/CommerceOrganizationClayTableDataSetDisplayView\.java" />
```

Comment:
@alecsloan im not sure about adding this suppression, is there reason why we dont just add that language key?

Reply:
we already have the language key in CE and it's usable, but SF doesn't seem to detect it. I mentioned to Hugo: https://liferay.slack.com/archives/C1SJ64S9X/p1600206039024800

Reply:
ahhh ic kk, sounds good to me. thanks!